doc/guides/modules.rst

```diff
@@ -510,6 +603,8 @@ Build settings supported in the :file:`module.yml` file are:
 - ``arch_root``: Contains additional architectures that are available to the
   build system. Additional architectures must be located in a
   :file:`<arch_root>/arch` folder.
+- ``module_root``: Contains :file:`CMakeLists.txt` and :file:`Kconfig` files for
```

Comment:
I think this needs a bit more explanation, perhaps in the Module root section.

Reply:
Not here, as this simply describes what additional roots that can be defined in the module.yml file.

Who those roots are used is described elsewhere, as you also suggest, in the Module root section.
Not sure what you want to describe though. Content of the CMakeLists.txt file and Kconfig file ?

There is no difference between a CMakeLists.txt or Kconfig file here or elsewhere in the build system.
Only difference, is that they are only included into the build system when the module is available.

So would be outside the scope of modules to describe the CMake or Kconfig content.

Reply:
I can't see that it's mentioned in the module root section. It just needs a sentence e.g. "Additional module roots can also be added in the module.yml file, see <ref>."

Reply:
ahh, now I get it :smile: .
Will do, thanks.

Reply:
done